src/languages/en.js

```diff
@@ -259,17 +259,26 @@ export default {
     },
     signInPage: {
         expensifyDotCash: 'Expensify.cash',
-        expensifyIsOpenSource: 'Expensify.cash is open source',
+        expensifyIsOpenSource: 'New Expensify is open source',
```

Comment:
@kevinksullivan is this how we want it to read? As stated elsewhere, this just seems clunky/grammatically incorrect to me.

Reply:
Perhaps `"The new Expensify is open source"`?